vllm/worker/hpu_model_runner.py

```diff
@@ -2308,10 +2312,6 @@ def profile_run(self) -> None:
             is_lora_profile_run=True,
         )
 
-        msg = (f"profiling run with {max_batch_size=}, {max_seq_len=}")
-        logger.info(msg)
-        self.warmup_scenario(max_batch_size, max_seq_len, True, kv_caches,
```

Comment:
why remove second self.warmup_scenario? can this warmup be covered with first one?

Reply:
Thanks for the clarification!
Should we call the first warmup only if `self.model_is_mrope == True`?

Reply:
Yes, we should call first warmup only if self.model_is_mrope == True.